README.md

```diff
@@ -283,6 +284,14 @@ cleaned up.
 </clean>
 ```
 
+#### Cleaning `catalog_category_product` entity
+A `catalog_category_product` entity isn't supported by Magento and therefore cleaning this entity works different then stated above. 
+`AvS_FastSimpleImport` supports three different types of import behaviours: 
+- Append Complex Data
+- Replace Existing Complex Data (default)
+- Delete Entities
```

Comment:
Dat stukje kan er denk ik wel helemaal uit, is meer iets vanuit `AvS_FastSimpleImport`.